pkg/rke/k8s_rke_system_images.go

```diff
@@ -10628,12 +10628,14 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.28.0",
 			CanalFlannel:              "rancher/mirrored-flannel-flannel:v0.25.1",
 			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.28.0",
-			AciCniDeployContainer:     "noiro/cnideploy:6.0.4.1.81c2369",
-			AciHostContainer:          "noiro/aci-containers-host:6.0.4.1.81c2369",
-			AciOpflexContainer:        "noiro/opflex:6.0.4.1.81c2369",
-			AciMcastContainer:         "noiro/opflex:6.0.4.1.81c2369",
-			AciOpenvSwitchContainer:   "noiro/openvswitch:6.0.4.1.81c2369",
-			AciControllerContainer:    "noiro/aci-containers-controller:6.0.4.1.81c2369",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
```

Comment:
We removed these Weave fields intentionally for v1.30, this is adding them back. Could you ensure we update only ACI fields for v1.30?

Reply:
Thanks for pointing this out, seem to have missed it while rebasing. Resolved it.